5-Accessibility/accessibility.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Accessibility</title>
+</head>
+<body>
+    <main>
+        <form action="">
+            <fieldset>
+                <legend><strong>Please enter your contact details</strong></legend>
+                <div>
+                    <label for="name"><strong>Your name (required):</strong> </label>
+                    <input type="text" id="name" name="user_name" required>
+                    <abbr title="required" aria-label="required"></abbr>
```

Comment:
nice to see the `aria-attributes`

Reply:
One thing here that I forgot. It's ok the way that you are using. But it's better to use the `aria-attribute` inside the `<input>`